### PR TITLE
UIEH-992: Add support for Settings (eHoldings): Can create, edit, and view knowledge base credentials.

### PR DESCRIPTION
**Pull request by Dmitriy-Litvinenko**
## Description
-  Add support for Settings (eHoldings): Can create, edit, and view knowledge base credentials.

## Issue
[UIEH-992](https://issues.folio.org/browse/UIEH-992)

## Screenshots
image

### CODE REVIEW

File: src/routes/settings-knowledge-base.js [hunk 1]
```diff
@@ -164,8 +169,14 @@ class SettingsKnowledgeBaseRoute extends Component {
     const {
       kbCredentials,
       intl,
+      stripes,
+      history,
     } = this.props;
 
+    if (!stripes.hasPerm('ui-eholdings.settings.kb')) {
+      history.push('/settings/eholdings/knowledge-base');
```
Comment (BogdanDenis) [inline src/routes/settings-knowledge-base.js:177]: If want to hide the third pane with Kb settings you should redirect to `/settings/eholdings`

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: src/components/settings/settings.js [hunk 2]
```diff
@@ -144,7 +144,16 @@ class Settings extends Component {
...
-    const { location: { pathname } } = this.props;
+    const {
+      location: {
+        pathname,
+      },
+      stripes,
+    } = this.props;
+
+    if (!stripes.hasPerm('ui-eholdings.settings.kb')) {
+      return (<span />);
```
[the remark this answers is not in the document]
Comment (Dmitriy-Litvinenko) [reply src/components/settings/settings.js]: Fixed

File: src/routes/settings-knowledge-base.js (hunk 1 above)
Comment (Dmitriy-Litvinenko) [reply src/routes/settings-knowledge-base.js]: Fixed

**Review by Vladyslav-Velytskyi — COMMENTED** [review]
I see that this task isn't a part of usage consolidation.
@BogdanDenis shouldn't this task merge into `master` instead of `settings-uc`?

**Review by Vladyslav-Velytskyi — APPROVED** [review]

**Review by BogdanDenis — APPROVED** [review]

**Pull request merged**